src/search/FindInFiles.js

```diff
@@ -734,67 +742,81 @@ define(function (require, exports, module) {
          * Remove existing search results that match the given entry's path
          * @param {(File|Directory)} entry
          */
-        function _removeSearchResultsForEntry(entry) {
-            Object.keys(searchModel.results).forEach(function (fullPath) {
-                if (fullPath === entry.fullPath ||
-                        (entry.isDirectory && fullPath.indexOf(entry.fullPath) === 0)) {
-                    // node search : inform node that the file is removed
-                    filesRemoved([fullPath]);
-                    if (findOrReplaceInProgress) {
-                        searchModel.removeResults(fullPath);
-                        resultsChanged = true;
+        function _removeSearchResultsForEntries(entries) {
+            var fullPaths = [];
+            entries.forEach(function (entry) {
+                Object.keys(searchModel.results).forEach(function (fullPath) {
+                    if (fullPath === entry.fullPath ||
+                            (entry.isDirectory && fullPath.indexOf(entry.fullPath) === 0)) {
+                        // node search : inform node that the file is removed
+                        fullPaths.push(fullPath);
+                        if (findOrReplaceInProgress) {
+                            searchModel.removeResults(fullPath);
```

Comment:
zaggino@ Bro how are you today? Please Bro help us we want CoffeeScript in Brakets and also  
Babel , and many's other please